decidim-core/lib/decidim/form_builder.rb

```diff
@@ -20,6 +20,7 @@ class FormBuilder < FoundationRailsHelper::FormBuilder
     # html_options - a Hash with options
     #
     # Renders a collection of check boxes.
+    # rubocop:disable Metrics/ParameterLists
```

Comment:
I really hate having to do this (I understand we can't help it since we're overriding a method, but...)